backup-postgres/action.yml

```diff
@@ -0,0 +1,69 @@
+name: Backup postgres
+description: Backup an Azure postgres database
+inputs:
+  storage-account:
+    description: Name of the Azure storage account to store the backup
+    type: string
+    required: true
+  app-name:
+    description: Name of the app deployment
+    type: string
+    required: true
+  resource-group:
+    description: Name of the Azure resource group
+    type: string
+    required: true
+  azure-credentials:
+    description: A JSON string containing service principle credentials.
+    required: true
+  environment:
+    description: App environment
+    required: true
+  backup-file:
+    description: Name of the backup file
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - uses: azure/login@v2
+      with:
+        creds: ${{ inputs.azure-credentials }}
+
+    - name: Setup postgres client
+      uses: DFE-Digital/github-actions/install-postgres-client@master
+      with:
+        version: 14
+
+    - name: Install kubectl
+      uses: DFE-Digital/github-actions/set-kubectl@master
+
+    - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
+      with:
+        azure-credentials: ${{ inputs.azure-credentials }}
+
+    - name: K8 setup
+      shell: bash
+      run: |
+        make ci ${{ inputs.environment }} get-cluster-credentials
```

Comment:
As discussed, we should remove the dependency on make commands

Reply:
removed dependency on make